LearningMissionLab/FinancialEvent.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class FinancialEvent
+    {
+        readonly Guid _financialId;
+        Dictionary<Guid, FinancialInfo> _financialAidPoolDictionary = new Dictionary<Guid, FinancialInfo>();
```

Comment:
These three dictionaries (_financialAidPoolDictionary, _salaryPoolDictionary, _stipendPoolDictionary) do not make any sense in the FinancialEvent class, they do not belong here, please remove. FinancialEvent describes just one event(record) in the person's financial history (FinancialInfo). It has the type, the start date, the end date, and the amount. It also has the id (GUID) field, the accountId field and the createDate and the updateDate fields, like any other record.